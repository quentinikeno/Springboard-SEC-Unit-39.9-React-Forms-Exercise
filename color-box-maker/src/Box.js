import React from "react";

const Box = (props) => {
	return <div style={props}></div>;
};

export default Box;
