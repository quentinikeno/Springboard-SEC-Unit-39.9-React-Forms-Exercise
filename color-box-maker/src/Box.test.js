import { render } from "@testing-library/react";
import Box from "./Box";

test("rendering without crashing", () => {
	render(<Box width={100} height={100} backgroundColor={"purple"} />);
});

test("matching a snapshot", () => {
	const { asFragment } = render(
		<Box width={100} height={100} backgroundColor={"purple"} />
	);
	expect(asFragment()).toMatchSnapshot();
});
