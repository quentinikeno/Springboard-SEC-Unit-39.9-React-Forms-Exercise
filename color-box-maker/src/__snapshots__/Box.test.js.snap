// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matching a snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      style="height: 100px; width: 100px; background-color: purple;"
    />
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
